tests/CMakeLists.txt

```diff
@@ -225,6 +225,14 @@ if(PYTHON_VERSION_MAJOR GREATER 2 AND SSG_PRODUCT_RHEL9)
     ssg_refcheck_test("rhel9" "cis" "cis")
     ssg_refcheck_test("rhel9" "ccn_basic" "ccn")
     ssg_refcheck_test("rhel9" "ccn_advanced" "ccn")
+    # This can moved back the macro once enable_authselect has a stigid
```

Comment:
Don't increase technical debt. I think it would be better to add a parameter to the macro instead of copying the macro code.